core-bundle/contao/widgets/ImageSize.php

```diff
@@ -240,12 +240,12 @@ public function generate()
 		}
 
 		return \sprintf(
-			'<div id="ctrl_%s" class="tl_image_size%s"%s>%s</div>%s',
+			'<div id="ctrl_%s" class="tl_image_size%s"%s>%s%s</div>',
 			$this->strId,
 			$this->strClass ? ' ' . $this->strClass : '',
 			$this->getStimulusAttributes($arrValues),
 			implode(' ', $arrFields),
-			$this->wizard
+			$this->wizard ?: '<button type="button" data-contao--tooltips-target="tooltip" data-contao--image-size-target="button" data-action="contao--image-size#openModal" disabled>' . Image::getHtml('edit--disabled.svg', attributes: ' data-contao--image-size-target="image"') . '</button>'
```

Comment:
```suggestion
			$this->wizard ? '': '<button type="button" data-contao--tooltips-target="tooltip" data-contao--image-size-target="button" data-action="contao--image-size#openModal" disabled>' . Image::getHtml('edit--disabled.svg', attributes: ' data-contao--image-size-target="image"') . '</button>'
```

Would output '1' if wizard is enabled, wouldn't it?